cogs/assistance-cmds/twl.3ds.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Fix broken TWL
-url: https://wiki.hacks.guide/wiki/3DS:TWLFix](https://wiki.hacks.guide/wiki/3DS:DS_mode_unbrick#Section_IV,_Part_I_-_Regular_TWLFix
```

Comment:
the TWLfix command is now pointing to the DSU page so this in specific is not needed